app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -11,12 +11,15 @@
 .gem-c-pagination__item {
   @include govuk-font($size: 16, $line-height: (20 / 16));
   list-style: none;
+  height: 70px;
+  border-top: 1px solid $govuk-border-colour;
+  margin-bottom: 16px;
```

Comment:
Why is this 16px? Can we not round this down to 15px and use the govuk-frontend spacing scale?

Reply:
Question has been raised with the designer

Reply:
Decision from Stephen McCarthy to go with the 15px and use the standard govuk-frontend spacing